### PR TITLE
Made changes to README

### PR DESCRIPTION
**Pull request by HyperSourceGithub**
- Added "System Requirements" section
- Added "Contributing" section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,21 +44,31 @@
 </p>
 </div>
 
+## System Requirements
+OS: Android 8.1 through 14 *(note: Android 14 may have some bugs with experimental features. want to help fix? check out the [contributing](https://github.com/dead8309/Kizzy?tab=readme-ov-file#Contributing) section!)* <br />
```
Comment (dead8309) [inline README.md:48]: Use bullet points

File: README.md [hunk 2]
```diff
@@ -106,6 +132,38 @@ However people have been using custom rich presence for past 4-5 years and there
 ## Translate
 See [Contribute for Translation](https://github.com/dead8309/Kizzy/issues/2#issue-1370365856)
 
+## Contributing
```
Comment (dead8309) [inline README.md:135]: I'll add the `Contributing.md` file later. No need for this as of now

File: README.md (hunk 1 above)
Comment (HyperSourceGithub) [reply README.md:48]: Sure, will update soon.

File: README.md (hunk 2 above)
Comment (HyperSourceGithub) [reply README.md:135]: Got it, thanks.

**Pull request merged**